shub/image/test.py

```diff
@@ -105,4 +109,5 @@ def _run_docker_command(client, image_name, command):
                        stderr=True if statuscode else False,
                        stream=False, timestamps=False)
     utils.debug_log("{} results:\n{}".format(command, logs))
+    client.remove_container(container)
```

Comment:
Good idea, but there's no analogue of `docker run` in `docker-py==1.10.6`, it appears only in [`docker-py==2.0`](https://github.com/docker/docker-py/blob/2.0.0/docker/models/containers.py#L422) version - and it requires testing. Considering above I'd go with a context manager for the PR